client/my-sites/domains/domain-management/list/list-all.jsx

```diff
@@ -91,6 +158,72 @@ class ListAll extends Component {
 		return ! hasAllSitesLoaded || ( requestingFlatDomains && domainsList.length === 0 );
 	}
 
+	isRequestingSiteDomains() {
+		const { requestingSiteDomains } = this.props;
+
+		return (
+			Object.keys( requestingSiteDomains ?? {} ).length === 0 ||
+			Object.values( requestingSiteDomains ).reduce( ( result, value ) => result || value, false )
```

Comment:
Are the values of `requestingSiteDomains` always boolean? If so, this is fine. If not, you'd need to cast the last `value`, but it looks like they may already be. I just wanted to check.

Reply:
Yes, these should always be booleans.